template/_.eslintrc.js

```diff
@@ -9,11 +9,17 @@ module.exports = {
     use: true
   },
   <%_ } _%>
+  <%_ if (!typescript) { _%>
   parserOptions: {
     parser: 'babel-eslint'
```

Comment:
Is there any incompatibility if we use babel-eslint parser? (asking because a project may use mixed js/ts. So having babel-eslint can be more consistent)

Reply:
Yes there is incompatibility cause this overrides https://github.com/nuxt/eslint-config/blob/master/packages/eslint-config-typescript/index.js#L7.

It needs ESLint config to not have top-level custom `parser`. There are also incompatibility if the user extends other configurations that may change the parser, and in this case, `@nuxtjs/eslint-config-typescript` needs to be at the end of the `extends` array.

The TS linter handles JS files as well so it's okay :)

Reply:
Good point

Reply:
https://github.com/babel/babel-eslint/issues/663

Reply:
I don't have any issue having my project linting JS files, I don't know about `jsx` files though which is more likely better handled by `babel`.

Reply:
![image](https://user-images.githubusercontent.com/25272043/70145938-7bc0cc80-16a1-11ea-8b97-4ea0d27004ef.png)